.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Hydra W3C Community Group
+
+This repository is being used for work in the Hydra W3C Community Group, governed by the
+[W3C Community License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/).
+To contribute, you must [join the Community Group](http://www.hydra-cg.com/#community).
+
+## How to contribute
+
+To advance Hydra, it is imperative that the community engages with us in discussing new and existing features. The only
+way to hope for an end result useful for as many people as possible is for the actual interested parties to collaborate
+throughout the process of building the Hydra's specification. This is the essence of an Open (Source) community.
+
+This document gathers some guidelines which should help make this collaboration as smooth as possible.
+These are not strict rules.
+
+### Getting in touch
+
+The best way to start contributing is to first spark some interest. This can be done by creating a GitHub issue
+(more on that below) but a quicker alternative is to reach out to the community through on of our channels mentioned below.
+
+#### Mailing list
+
+The more traditional, and traceable method is to sign up and contact us via the [mailing list][ml]. All messages
+are stored in an archive and can be used as reference in the future.
+
+[ml]: https://lists.w3.org/Archives/Public/public-hydra/
+
+#### Slack
+
+As means of instant communication we're monitoring a `#hydra` channel on the HTTP APIs Slack. To register, please
+go to http://slack.httpapis.com.
+
+#### Twitter
+
+You can also Tweet to [`@HydraCg`](https://twitter.com/hydracg)
```

Comment:
Who is controlling / monitoring this account?

Reply:
Markus owns it. I'm also added to Tweetdeck. Maybe I could add a Zapier zap to notify `@hydra-cg` on Slack? Otherwise it's easy to miss that something shows up

Reply:
Let's put this to the agenda of our next call.